datafusion/src/physical_plan/parquet.rs

```diff
@@ -72,6 +76,8 @@ pub struct ParquetExec {
     batch_size: usize,
     /// Statistics for the data set (sum of statistics for all partitions)
     statistics: Statistics,
+    /// Numer of times the pruning predicate could not be created
+    predicate_creation_errors: Arc<SQLMetric>,
```

Comment:
would it make sense to move `predicate_creation_errors` into `ParquetMetrics`, next to the other two metrics?

Reply:
The reason I didn't do this is because the other metrics are per `ParquetPartition` (aka per each file / set of files) but this one is just one metric for the overall `ParquetExec`

I agree this is confusing -- I'll make two statistics structures (`ParquetPartitionMetrics` and `ParquetMetrics`) which hopefully will make this clearer

Reply:
in 595fcdd07b